isisd/isis_bfd.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * IS-IS Rout(e)ing protocol - BFD support
+ *
+ * Copyright (C) 2018 Christian Franke
+ *
+ * This file is part of FreeRangeRouting (FRR)
```

Comment:
again let's use the right template here

Reply:
Where can I find that template?

Reply:
https://github.com/FRRouting/frr/blob/master/doc/developer/workflow.rst